### ISSUE TITLE
Feature Request: Improve Lighthouse SEO Score

### ISSUE THREAD

**Issue by wordshaker** [DDDEastMidlandsLimited/dddem-web#215]
### Describe the feature

The site currently has a score of 77 for SEO when Lighthouse is run against it. 

The details of improvements that can be made are shown in the lighthouse report

![image](https://user-images.githubusercontent.com/13496774/110368060-6b379880-8040-11eb-8fa7-2b700d18c1cf.png)

### To Get Lighthouse Score

- Open the site in Chrome
- Open developer tools
- Go to the Lighthouse tab in the dev tools
- Run.

### Advantages

- Improving the SEO of the site

#### Acceptance Criteria

Functional and non-functional criteria that should be met by this work

- [ ] Lighthouse score for SEO is increased to at least 90.

**Comment by wordshaker** [comment]
![image](https://user-images.githubusercontent.com/13496774/111886168-1c84e980-89c4-11eb-972c-e94be93cbdbc.png)

Not quite at 90 yet

**Comment by wordshaker** [comment]
Just saw the comment on the pull request so we will do something about the links. Thanks @paolapog and @johnnolan
